pages/assessment.tsx

```diff
@@ -50,10 +50,14 @@ const Assessment: NextPage = () => {
     'let - block scope. Dersom variabelen blir deklarert med let i en funksjon, er den bare tilgjengelig i funksjonen. var - global scope Dersom variabelen blir deklarert med var, blir den tilgjengelig i all kode. Kan by på problemer når vi gir variabler samme navn.';
```

Comment:
All prettier except for line 176

Reply:
I really cannot find this change! 😬  I rebased, and now there is nothing that i can see.
I have looked at the rebase branches 3b59f38 and da2547c. one of them should contain your code before I rebased 🙂